src/parcel_snoopi_deep.jl

```diff
@@ -905,8 +911,8 @@ The empty horizontal periods in the flamegraph correspond to times when somethin
 The total width of the flamegraph is set from the `ROOT` node.
 """
 function FlameGraphs.flamegraph(tinf::InferenceTimingNode; tmin = 0.0, excluded_modules=Set([Main::Module]), mode=nothing)
+    isROOT(tinf) && isempty(tinf.children) && error("root node has no children")
```

Comment:
@timholy I just hit this error in usage of this package..

It seems to me that it's much better to just return an otherwise empty profile, with just the single ROOT node.
From what i can tell, without this check, it _does_ return that empty profile correctly.

Can we remove this line? I'm curious if you remember the motivation to add it.

Reply:
Very happy to have you change this though!

Reply:
Yeah, makes sense. I think i was thinking that if it's empty, then it's empty! 😅  But you're right that it's not as nice as if it said "hey this is empty."

Maybe we can print a warning instead of throwing an error?

Our use case is that we're creating this via an HTTP endpoint on prod servers now (!!), and i don't want that to throw an error and return a 500:
https://github.com/JuliaPerf/ProfileEndpoints.jl/pull/16

K, thanks, i'll send a quick PR